static/js/app.js

```diff
@@ -11,14 +11,15 @@ require('nvd3');
 
 require('angular');
 require('angular-cookie');
+require('angular-pouchdb');
```

Comment:
are these extra requires necessary? The code change below looks like it's just using `$translateMessageFormatInterpolation`.